### PR TITLE
Add default Laravel migrations

### PR DESCRIPTION
**Pull request by Jeroen-G**
Seeing that Laravel comes with these migrations by default, I assumed it would be appropriate as well if Testbench includes them, so every package does not have to include them for testing purposes.

If you want me to make a note in the changelog, let me know.

If you want me to PR this to any other branch, let me know. The files were in Laravel for a couple versions, so I suppose I could send a PR for those as well.

### CODE REVIEW

**Comment by Jeroen-G** [comment]
I don't see how Travis could fail, my tests passed...

**Comment by crynobone** [comment]
I have plan to add this but not on the default Laravel path.

- Use `fixture/migrations` folder.
- add `TestCase::runLaravelMigration()` with execute migration with `--path` to the proper folder.
- update the read me.

**Comment by crynobone** [comment]
And this should go s low as `3.1` branch.

**Comment by Jeroen-G** [comment]
Why not in the default place?

**Comment by crynobone** [comment]
That a major breaking change. Not going to ever do that.

**Comment by Jeroen-G** [comment]
Ok, but how is it going to break? I'm not being critical, just curious. All I did was placing the files in the existing directory, and nothing broke when I tried it.

**Comment by crynobone** [comment]
Certain packages especially those that build own user management might have different `users` migrations and this is working fine on Testbench all this while. 

By adding these migration to the default. There no easy way to avoid them to be executed on Testbench.

**Comment by Jeroen-G** [comment]
Got it, will work on it.

**Comment by Jeroen-G** [comment]
So, it works just like you'd like to see it. Only problem is (and this is why Travis failed), there now exist two CreateUsersTable classes in the codebase. One in fixture/migrations and one in tests/migrations.
If you are okay with it, I would change the latter into something else than users.

**Comment by crynobone** [comment]
There's no need to change that. Just follow exactly as https://github.com/orchestral/testbench/pull/157#issuecomment-279183695

**Comment by Jeroen-G** [comment]
You must get tired of people having issues with migrations ^_^

Have a look at the commits, and the outcome of the Travis build. I'm pretty sure if I after this push a commit which changes the name of the migration in tests/migration to `CreateTestbenchUsersTable` it would pass.

File: src/TestCase.php [hunk 1]
```diff
@@ -198,4 +198,26 @@ protected function getEnvironmentSetUp($app)
...
+    /**
+     * Migrate Laravel's default migrations.
+     * 
+     * @param  string $database
+     * 
+     * @return void
+     */
+    public function runLaravelDefaultMigrations($database = null)
+    {
+        $options = ['--path' => '../fixture/migrations'];
```
Comment (crynobone) [inline src/TestCase.php]: Shouldn't this be just `$options = ['--path' => 'migrations'];`

https://github.com/laravel/framework/blob/5.4/src/Illuminate/Database/Console/Migrations/BaseCommand.php#L20

Comment (Jeroen-G) [inline src/TestCase.php]: Tried it and yes, that works as well. Will change it.
However, it does not solve the `cannot redeclare class` problem, which a rename of `tests/migrations/2013_07_26_182750_create_users_table.php` would.

Comment (Jeroen-G) [inline src/TestCase.php]: You merged before I could commit :P

Comment (crynobone) [inline src/TestCase.php]: Nevermind, I need to work on backporting this.

Comment (Jeroen-G) [inline src/TestCase.php]: The commit for this and the file renaming are on my branch if you need them.
As long as the file isn't renamed, this package's tests will fail.

**Pull request merged**